File: Lista1 - py/Lista1casa.py
#R = float(input("Quantos reais você quer converter para dolar:"))     #Exercicio 1
#T = R/2.4
#print(T)

#D = float(input("Quantos dolares você vai converter em real:"))       #Exercicio 2 
#T2 = D * 2.4
#print(T2)

#AP = float(input("Qual a altura da parede:"))                              #Exercicio 3
#LP = float(input("Qual a largura da parede:"))                               
#AA = float(input("Qual a altura do azulejo:"))
#LA = float(input("Qual a largura do azulejo:"))
#T = (AP * LP) / (AA * LA)
#print(f"Vai precisa de {T} azulejos")

#h = float(input("Qual a altura do retângulo:"))                        #Exercicio 4
#c = float(input("Qual a comprimento do retângulo:"))
#Area = h * c
#Per = 2 * h + 2 * c
#print(f"Esse é a sua area:{Area}")
#print(f"Esse é o seu perimetro:{Per}")

#M = float(input("Qual a sua massa:"))                                  #Exercicio 5
#H = float(input("Qual a sua altura:"))
#IMC = M / H**2
#print(IMC)

import math

#r = float(input("Qual o valor do raio: "))                             #Exercicio 6
#A = r ** 2 * math.pi
#C = 2 * r * math.pi
#print(f"A sua área é: {A}, e sua comprimento é: {C}")

#R = float(input("Qual o valor do seu raio:"))                          #Exercicio 7
#V = 4/3 * R ** 3 * math.pi
#A = 4 * math.pi * R ** 2
#print(f"Seu volume é: {V}, e sua area é: {A}") 

#pb = float(input("Qual foi sua nota no 1 bimestre:"))                  #Exercicio 8
#sb = float(input("Qual foi sua nota no 2 bimestre:"))
#tb = float(input("Qual foi sua nota no 3 bimestre:"))
#qb = float(input("Qual foi sua nota no 4 bimestre:"))
#NF = (pb + sb + tb + qb)/4
#print(NF)

#P1 = float(input("Qual foi a nota do aluno no 1 semestre:"))           #Exercicio 9
#P2 = float(input("Qual foi a nota do aluno no 2 semestre:"))
#AT = float(input("Qual foi a nota do aluno na atividade:"))
#M = (P1 * 4 + P2 * 4 + AT * 2) /10
#print("Sua nota foi", M)

#a = float(input("Valor de a:"))                                        #Exercicio 10
#b = float(input("Valor de b:"))
#c = a
#a = b
#b = c
#print(f"O valor de a é:{a}\nO valor de b é: {b}")

#a = float(input("Valor de a:"))                                        #Exercicio 11                                   #Exercicio 10
#b = float(input("Valor de b:"))
#a,b = b,a 
#print(f"O valor de a é:{a}\nO valor de b é: {b}")

#D = float(input("Qual foi sua distância:"))                            #Exercicio 12
#T = float(input("Qual foi seu tempo de gasto:"))
#VM = D/T
#print(f"Sua velocidade media foi {VM}")

t = int(input("Qual foi o valor de tempo:"))                           #Exercicio 13
s0 = 2
v0 = 3
a = 10
S = s0 + v0 * t + 1/2 * a * t ** 2
print(f"O resultado é:{S}")

File: Lista1 - C/Lista1casa.c
#include <stdio.h>
#include <math.h>
int main(){
    
    /*
    float R;                                        
    float D;
    printf("Qual o valor em reais:");
    scanf("%f", &R);
    D = R/2.4;
    printf("Valor em dolares é:  %.2f", D);
    return 0;
    */
    
    /*
    float R, D;                                     
    printf("Qual o seu valor em dolares:"); 
    scanf("%f", &D);
    R = D * 2.4;
    printf("Seu valor em reais é: %.2f", R);
    */
    /*
    float T, AP, LP, AA, LA;
    printf("Qual é a altura da parede: ");
    scanf("%f", &AP);
    printf("Qual é a largura da parede: ");
    scanf("%f", &LP);
    printf("Qual é a altura do azulejo: ");
    scanf("%f", &AA);
    printf("Qual é a largura do azulejo: ");
    scanf("%f", &LA);
    T = (AP * LP) / (LA * AA);
    printf("O total de azulejo necessario será: %.2f", T);
    */
    /*
    float h, c, A, P;
    printf("Qual a altura do retângulo: ");
    scanf("%f", &h);
    printf("Qual o comprimento do rentângulo: ");
    scanf("%f", &c);
    A = h * c;
    P = 2 * h + 2 * c;
    printf("Esse é a sua area: %.2f", A);
    printf("E Esse o seu perimetreo: %.2f", P);
    */
    /*
    float M, H, IMC;
    printf("Qual sua massa: ");
    scanf("%f", &M);
    printf("Qual a sua altura: ");
    scanf("%f", &H);
    IMC = M / pow(H, 2);
    printf("Seu IMC será: %.5f", IMC);
    */
    /*
    float r, A, C;
    printf("Qual o valor do raio: ");
    scanf("%f", &r);
    A = pow(r, 2) * 3.1415;
    C = r * 2 * 3.1415;
    printf("Sua area é: %.2f", A);
    printf("E seu perimetro é: %.2f", C);
    */
    /*
    float R, V, A;
    printf("Qual o valor do seu raio: ");
    scanf("%f", &R);
    V = 1.333 * pow(R, 3) * 3.1415;
    A = 4 * 3.1415 * pow(R, 2);
    printf("Seu volume é: %.2f", V);
    printf("Sua area é: %.2f", A);
    */
    /*
    float sb, pb, tb, qb, NF;
    printf("Qual foi sua nota no 1 bimestre: ");
    scanf("%f", &pb);
    printf("Qual foi sua nota no 2 bimestre: ");
    scanf("%f", &sb);
    printf("Qual foi sua nota no 3 bimestre: ");
    scanf("%f", &tb);
    printf("Qual foi sua nota no 4 bimestre: ");
    scanf("%f", &qb);
    NF = (pb + sb + tb + qb)/4;
    printf("Sua nota foi: %.2f", NF);
    */
    /*
    float P1, P2, AT, M;
    printf("Qual foi a nota no 1 semestre: ");
    scanf("%f", &P1);
    printf("Qual foi a nota no 2 semestre: ");
    scanf("%f", &P2);
    printf("Qual foi a nota do aluno na atividade: ");
    scanf("%f", &AT);
    M = (P1 * 4 + P2 * 4 + AT * 2)/10;
    printf("Sua nota foi: %.2f", M);
    */
    /*
    float a, b, c;
    printf("Valor de a: ");
    scanf("%f", &a);
    printf("Valor de b: ");
    scanf("%f", &b);
    c = a;
    a = b;
    b = c;
    printf("O valor de a virou: %.2f \n", a);
    printf("O valor de b virou: %.2f \n", b);
    */
    /*
    float a, b;
    printf("Valor de a: ");
    scanf("%f", &a);
    printf("Valor de b: ");
    scanf("%f", &b);
    a, b = b, a;
    printf("O valor de a é: %.2f\n", a);
    printf("O valor de b é: %.2f\n", b);
    */
    /*
    float d, t, vm;
    printf("Qual foi sua distância: ");
    scanf("%f", &d);
    printf("Qual foi seu tempo gasto: ");
    scanf("%f", &t);
    vm = d/t;
    printf("Sua velocidade média foi: %.2f", vm);
    */
    /*
    float t, s;
    printf("Qual foi o valor de tempo: ");
    scanf("%f", &t);
    s = 2 + 3 * t + 0.5 * 10 * pow(t, 2);
    printf("O resultado é: %.2f", s);
    */
}